connect/api/src/main/java/org/apache/kafka/connect/source/SourceConnector.java

```diff
@@ -24,4 +24,11 @@
  */
 public abstract class SourceConnector extends Connector {
 
+    /**
+     * {@inheritDoc}
+     */
```

Comment:
```suggestion
```
we are not adding anything to the javadoc of the super method, so we shouldn't need `@inheritDoc`